src/replication/adapter/harbor/image_registry.go

```diff
@@ -168,3 +168,10 @@ func (a *adapter) listArtifacts(repository string, filters []*model.Filter) ([]*
 	}
 	return filter.DoFilterArtifacts(arts, filters)
 }
+
+func (a *adapter) DeleteTag(repository, tag string) error {
+	project, repository := utils.ParseRepository(repository)
+	url := fmt.Sprintf("%s/api/%s/projects/%s/repositories/%s/artifacts/%s/tags/%s",
```

Comment:
This is not a good practice, it should be defined in a unified model linked with the API version.

Something like:
   `v2().Tag(t string).URL()` etc.